pkg/restapi/didmethod/operation/models.go

```diff
@@ -50,9 +50,11 @@ type Key struct {
 
 // PublicKey public key
 type PublicKey struct {
-	ID    string `json:"id,omitempty"`
-	Type  string `json:"type,omitempty"`
-	Value string `json:"value,omitempty"`
+	ID       string   `json:"id,omitempty"`
+	Type     string   `json:"type,omitempty"`
+	Value    string   `json:"value,omitempty"`
+	Usage    []string `json:"usage,omitempty"`
+	Encoding string   `json:"encoding,omitempty"`
```

Comment:
Make sure that you send it to Sidetree in JWK format: 
    {
      "id": "dual-key",
      "type": "JwsVerificationKey2020",
      "usage": ["auth", "general"],
      "publicKeyJwk": {
        "kty": "EC",
        "crv": "P-256K",
        "x": "PUymIqdtF_qxaAqPABSw-C-owT1KYYQbsMKFM-L9fJA",
        "y": "nM84jDHCMOTGTh_ZdHq4dBBdo4Z5PkEOW9jA8z8IsGc"
      }

Reply:
ok